main.go

```diff
@@ -98,6 +98,20 @@ func main() {
 			Name:  "no-cacerts",
 			Usage: "Skip CA certs population in settings when set to true",
 		},
+		cli.StringFlag{
+			Name:  "audit-log-path",
+			Usage: "Specifies the log path for Rancher Server API, not specifying means diasble server API audit log",
+		},
+		cli.IntFlag{
+			Name:  "audit-log-limit-size",
+			Value: 2048,
+			Usage: "Specified total size in megabytes all audit log files could use, audit log file will rotate each 100M.",
```

Comment:
`Limits total size in megabytes all audit log files could use. The file will rotate after it reaches 100M.`